draft-ietf-rtcweb-jsep.xml

```diff
@@ -2343,24 +2338,25 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <list style="symbols">
 
             <t>The m= line and corresponding "a=rtpmap" and "a=fmtp"
-            lines MUST only include codecs present in the most recent
-            answer which have not been excluded by the codec preferences
-            of the associated transceiver. Note that non-JSEP endpoints
-            are not subject to these restrictions, and might offer media
-            formats that were not present in the most recent answer, as
-            specified in <xref target="RFC3264"></xref>, Section 8.
-            Therefore, JSEP implementations MUST be prepared to receive such
-            offers.</t>
+            lines MUST only include media formats which have not been excluded
+            by the codec preferences of the associated transceiver, and
+            MUST include all currently available formats. Media formats that
+            were previously offered but are no longer available (e.g., a
+            shared hardware codec) MAY be excluded.</t>
 
             <t>Unless codec preferences have been set for the associated
             transceiver, the media formats on the m= line MUST be generated in
-            the same order as in the current local description.</t>
+            the same order as in the most recent answer. Any new media
```

Comment:
nit: "new" isn't quite the right word, these codecs may have been in previous offers